### PR TITLE
Add color

### PR DESCRIPTION
**Pull request by AbdulrahmanTabbakh**

### CODE REVIEW

File: src/Components/ColorForm/ColorForm.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <label>
+        Rolle:
+        {}
+        <input
+          type="text"
+          value={role}
+          onChange={(Event) => setRole(Event.target.value)}
```
Comment (ghost) [inline src/Components/ColorForm/ColorForm.jsx]: The standard way to write event is:
onChange={(event) => setRole(event.target.value)} **or** onChange={(e) => setRole(e.target.value)}

File: src/Components/ColorInput/ColorInput.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export default function ColorInput({ label, value, onChange }) {
+  const handleChange = (Event) => {
```
Comment (Rahaf91) [inline src/Components/ColorInput/ColorInput.jsx:2]: also here write (event)

File: src/Components/ColorInput/ColorInput.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    onChange(Event.target.value);
+  };
```
Comment (ghost) [inline src/Components/ColorInput/ColorInput.jsx:4]: also here write (event)

File: src/Components/ColorForm/ColorForm.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+    setRole("Primary");
+    setHex("#000000");
+    setContrastText("#ffffff");
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <label>
+        Rolle:
```
Comment (ghost) [inline src/Components/ColorForm/ColorForm.jsx]: small mistake:  Role

File: src/App.jsx [hunk 5]
```diff
@@ -1,15 +1,24 @@
 import { initialColors } from "./lib/colors";
-import Color from "./Components/Color/Color";
+
 import "./App.css";
+import { useState } from "react";
+import ColorForm from "./Components/ColorForm/ColorForm.jsx";
+import Color from "./Components/Color/Color.jsx";
```
Comment (ghost) [inline src/App.jsx:6]: you don't need to specify the file extension (react conventions)

import ColorForm from "./Components/ColorForm/ColorForm";
import Color from "./Components/Color/Color";

File: src/Components/ColorForm/ColorForm.jsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+import { nanoid } from "nanoid";
+import ColorInput from "../ColorInput/ColorInput.jsx";
+
+export default function ColorForm({ onAddColor }) {
+  const [role, setRole] = useState("Primary");
+  const [hex, setHex] = useState("#000000");
+  const [contrastText, setContrastText] = useState("#ffffff");
+
+  const handleSubmit = (Event) => {
+    Event.preventDefault();
```
Comment (ghost) [inline src/Components/ColorForm/ColorForm.jsx]: event:)

File: src/Components/ColorForm/ColorForm.jsx (hunk 4 above)
Comment (AbdulrahmanTabbakh) [reply src/Components/ColorForm/ColorForm.jsx]: Ok thanx i change it :)

**Pull request merged**